Scripts/utils/freight_costs.py

```diff
@@ -113,17 +102,27 @@ def get_aux_cost(unit_costs: Dict[str, Dict],
         Freight mode (truck/freight_train/ship)
     impedance : Dict[str, numpy.ndarray]
         Type (time/dist/toll) : numpy 2d matrix
-    impedance_aux : Dict[str, numpy.ndarray]
-        Auxiliary specific Type (time/dist/toll) : numpy 2d matrix
 
     Returns
     ----------
     auxiliary road cost : numpy.ndarray
         impedance type cost : numpy 2d matrix
     """
+    aux_dist = impedance["aux_dist"]
+    impedance["dist"] = impedance[f"aux_dist"]
+    del impedance[f"aux_dist"]
+    try:
+        impedance["time"] = impedance[f"aux_time"]
+        del impedance[f"aux_time"]
+    except KeyError:
+        pass
+    try:
+        del impedance["channel"]
+    except KeyError:
+        pass
```

Comment:
Modifying an existing dict can have unpredictable effects outside the function. I suggest creating a new dict:
```suggestion
    impedance_aux = {
        "dist": impedance["aux_dist"],
        "time": impedance["aux_time"],
    }
```

Why would "aux_time" not exist?